### DISCUSSION TITLE
varustust osta

### DISCUSSION THREAD

**Discussion by LisaWeiss** [ValveSoftware/Dota2-Gameplay#16969] [General]
Tere. Kust sa oma varustuse ostsid? näiteks arvuti või sülearvuti?

**Comment by MaxFolls** [comment]
Kõik sõltub sellest, mida täpselt otsite. Kui vajate kvaliteetset [Sülearvutid](https://avamarket.ee/arvutitehnika-b%C3%BCrootarbed/s%C3%BClearvutid-ja-tarvikud/s%C3%BClearvutid), siis on kõige parem võtta ühendust Avamarketi kauplusega. See on kõige õigem. Valik on seal väga suur, igaüks saab sealt endale sobiva variandi osta. Ja hea hinnaga ka. Ostsin ka sealt palju, siiani on kõik korras.
